CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Correct the generated path for downloading plugins by their names on Windows ([#4953](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4953))
 - [BUG] Fix buildPointSeriesData unit test fails due to local timezone ([#4992](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4992))
 - [BUG][Data Explorer][Discover] Fix total hits issue for no time based data ([#5087](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5087))
+- [BUG] ValidationError when setting maps.proxyOpenSearchMapsServiceInMaps ([#5122](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5122))
```

Comment:
Changelog entries should start with a verb and describe the impact of the change. And in the OpenSearch Project, by convention we link the PR (this PR) rather than the issue fixed. This would be something like:

```suggestion
- [BUG] Fix `maps.proxyOpenSearchMapsServiceInMaps` config definition so it can be set ([#5170](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5170))
```

Reply:
I manually made the change.